content/collections/data/en/data-backfill.md

```diff
@@ -49,8 +49,7 @@ This is Amplitude's recommendation for backfilling large amounts of data:
     2. Partition those events into requests based on `device_id` or `user_id`.
     3. Send your requests concurrently/in parallel to Amplitude.
 
-To optimize this process further, add aggressive retry logic with high timeouts. Continue to retry until you receive a 200 response. If you send an `insert_id`, 
-then Amplitude deduplicates data that has the same `insert_id` sent within 7 days of each other. 
+To optimize this process further, add aggressive retry logic with high timeouts. Continue to retry until you receive a 200 response. If you send an `insert_id`, then Amplitude deduplicates data that has the same `insert_id` sent within 7 days of each other. 
```

Comment:
<img width="793" alt="Screenshot 2024-12-20 at 4 17 08 PM" src="https://github.com/user-attachments/assets/85f5fbbd-456c-45aa-8303-1e3543a0cf4c" />

this had a weird line break

Reply:
Yeah, I've noticed that from time to time. It's hard to search for all of them. Thanks for fixing this one!